backend/resources/voucher.py

```diff
@@ -68,7 +69,7 @@ def voucher_post_institution(session, user_inst):  # pylint:disable=unused-argum
     voucher_price = request.headers.get('price', default=None)
     voucher_description = request.headers.get('subject', default=None)
     voucher_title = request.headers.get('title', default=None)
-    voucher_valid_time = request.headers.get('validTime', default=2 * 31536000)
+    voucher_valid_time = request.headers.get('validTime', default=1597847789)
```

Comment:
y?

1597847789 entspricht 19.08.2020
2 * 31536000 entspricht 2 Jahren (in Sekunden)

"validTime - Zeit, für die ein Gutschein gültig sein soll, in Sekunden" - API-Spec

Reply:
Mein Fehler, bin durcheinander gekommen und dachte irgendwie das man den Tag angibt an dem es abläuft.. Macht aber echt nicht so sinn